### PR TITLE
Fix description and units in derivative functions IdealGases.Common.Functions.{cp_Tlow_der, h_T_der, h_Tlow_der}

### PR DESCRIPTION
**Pull request by beutlich**
s0_Tlow_der is separately considered by #2919.

### CODE REVIEW

File: Modelica/Media/IdealGases/Common/Functions.mo [hunk 1]
```diff
@@ -94,8 +94,8 @@ package Functions
       "Choice of reference enthalpy";
     input SI.SpecificEnthalpy h_off=h_offset
       "User defined offset for reference enthalpy, if referenceChoice = UserDefined";
-    input Real dT "Temperature derivative";
-    output Real h_der "Specific enthalpy at temperature T";
+    input Real dT(unit="K/s") "Temperature derivative";
+    output Real h_der(unit="J/(K.s)") "Derivative of specific enthalpy at temperature T";
```
Comment (casella) [inline Modelica/Media/IdealGases/Common/Functions.mo]: The unit of h_der should be "J/(kg.s)", not "J/(K.s)"

**Comment by beutlich** [comment]
> Sounds good, except for the issue at line 98

Right. Resolved.

**Comment by beutlich** [comment]
@casella Please update your review.
@hubertus65 @wischhusen Please add your review.

**Comment by beutlich** [comment]
> 
> 
> @casella Please update your review.
> @hubertus65 @wischhusen Please add your review.

Please do not forget to add or update your review!

**Pull request merged**